### ISSUE TITLE
NB 01: Change loss type

### ISSUE THREAD

**Issue by magitz** [PracticumAI/computer_vision#18]
Need to update default loss in section 5 to be: `loss=SparseCategoricalCrossentropy(from_logits=True)`

Probably also used in NB 02.1

**Comment by ilutticken** [comment]
Fixed the issue. Change is on the Ians_Sep_2024_Edits branch.

**Comment by ilutticken** [comment]
This is ready to close if @magitz has merged the Ians_Sept_2024_Edits branch.

**Comment by magitz** [comment]
Merged. Closing.
